### PR TITLE
[Scalatra] Updated the version of Scalatra to the latest (2.6.2)

### PR DESCRIPTION
**Pull request by magnolia-k**
- Updated Scalatra 2.6.2 (latest)
 - Updated Scala 2.12.4
 - Updated sbt 1.1.0
 - Reduced dependent artifacts to a minimum
 - Cleaned up build.sbt
 - Changed the package name from client to server
 - Set log output (logback)
 - Updated web.xml for servlet 3.1

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Update for Scalatra 2.6.2

Please review.
@clasnake  @jimschubert  @shijinkui

### CODE REVIEW

**Comment by magnolia-k** [comment]
Thank you for the quick review.

Although I am going to change the package name, changing the package name of the following files is not reflected in the package names of every place (eg api).

```
modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ScalatraServerCodegen.java
```

Can you tell me the other places to fix?

**Comment by magnolia-k** [comment]
Once again, I made it!

File: modules/swagger-codegen/src/main/resources/scalatra/logback.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<configuration>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```
Comment (shijinkui) [inline modules/swagger-codegen/src/main/resources/scalatra/logback.xml]: Suggest also to add rolling file appender, it's more useful.

**Comment by shijinkui** [comment]
LGTM.
We can add java and scala checkstyle in another thread.

File: modules/swagger-codegen/src/main/resources/scalatra/logback.xml (hunk 1 above)
Comment (magnolia-k) [reply modules/swagger-codegen/src/main/resources/scalatra/logback.xml]: Thanks!

I added.

**Comment by wing328** [comment]
@magnolia-k thanks for the PR and the quick update based on the feedback.

Thanks @jimschubert @shijinkui for the quick review.

**Pull request merged**